api/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# RedHat UBI 8 with nodejs 16
-FROM registry.access.redhat.com/ubi8/nodejs-18:1-86 AS builder
+# RedHat UBI 8 with nodejs 20
+FROM registry.access.redhat.com/ubi8/nodejs-20:1-58 AS builder
```

Comment:
Hi @basilv , just FYI.
I update runtime for FOM to be run under nodejs 20. nodejs 18 will not be supported 2025.
The rest of the pr changes are not important, just some packages update this time and re npm install for all components(package-lock.json).